src/upstream.c

```diff
@@ -190,6 +190,16 @@ void *handle_upstream()
                     ParodusError("failure in allocation for message\n");
                 }
             }
+	    if(nn_shutdown(sock, bind) < 0)
```

Comment:
This is just for clean up on parodus shutdown right?

Reply:
Yes